server/proxy/pf_rdpgfx.c

```diff
@@ -241,7 +401,7 @@ static UINT pf_rdpgfx_caps_advertise(RdpgfxServerContext* context,
 		 * TODO: Have a better way of expressing max supported GFX caps version
 		 * by FreeRDP.
 		 */
-		if (currentCaps->version <= RDPGFX_CAPVERSION_106)
+		if (currentCaps->version <= RDPGFX_CAPVERSION_102)
```

Comment:
should revert, here by mistake (was testing something)

Reply:
ok, I´ll wait for the revert before merging ^^

Reply:
reverted :)
i'll be glad if while reviewing, you'll take a deeper look in pf_capture.c (i want to make sure that there are no problems with how the path is built).